storio-sqlitedb/src/main/java/com/pushtorefresh/storio/sqlitedb/operation/get/PreparedGetListOfObjects.java

```diff
@@ -181,7 +182,8 @@ public void call(Subscriber<? super List<T>> subscriber) {
         }
 
         /**
-         * Specifies {@link RawQuery} for Get Operation, you can use it for "joins" and same constructions which are not allowed in {@link Query}
+         * Specifies {@link RawQuery} for Get Operation,
```

Comment:
Hm, for this case `query` or `rawQuery` is required, I think we should notice that in javadoc